scene/resources/tile_set.cpp

```diff
@@ -619,16 +621,119 @@ Vector2 TileSet::autotile_get_subtile_for_bitmask(int p_id, uint16_t p_bitmask,
 		mask_ignore = mask >> 16;
 
 		if (((mask_ & (~mask_ignore)) == (p_bitmask & (~mask_ignore))) && (((~mask_) | mask_ignore) == ((~p_bitmask) | mask_ignore))) {
-			uint32_t priority = autotile_get_subtile_priority(p_id, E->key());
-			priority_sum += priority;
-			priorities.push_back(priority);
 			coords.push_back(E->key());
 		}
 	}
+	return coords;
+}
+
+uint32_t TileSet::_count_bitmask_bits(uint32_t bitmask) {
+	uint32_t ret = 0;
+
+	for (uint32_t i = 1; i <= 256; i <<= 1) {
+		if (bitmask & i) {
+			ret++;
+		}
+	}
+
+	return ret;
+}
+uint32_t TileSet::_score_bitmask_difference(uint32_t bitmask, uint32_t ref_bitmask) {
+	// low = less difference, high = more difference
+	uint32_t ret = 0;
+
+	bitmask ^= ref_bitmask;
+	// add one to the score for each non-matching bit
```

Comment:
```suggestion
	// Add one to the score for each non-matching bit.
```
Comments should start with capital letter and end with period.